### PR TITLE
#118 create hr home layout

### PR DESCRIPTION
**Pull request by MiroslawChmielecki**

### CODE REVIEW

File: src/styles/stylesForForms.css [hunk 1]
```diff
@@ -31,4 +31,72 @@
...
+    left: 100px;
+}
+
+.starNumber4 {
+    left: 150px;
+}
+
+.starNumber5 {
+    left: 200px;
+}
```
Comment (mae1579) [inline src/styles/stylesForForms.css]: Great css content

File: src/Validations/StudentFilteredValidation.ts [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+            )
+            .max(6, "Za długa wartość"),
+        monthsOfCommercialExp: yup
+            .string()
+            .nullable()
+            .test('correct-data', "Podaj liczbę całkowitą",
+                (val) => val !== null && val !== undefined && !(val.includes('.')) && !(val.includes(','))
+            )
+            .max(3, "Za długa wartość"),
+    })
```
Comment (patrycja-dev41171) [inline src/Validations/StudentFilteredValidation.ts:27]: Good use of validation.

File: src/components/StyledComponents/MainStyledTextField.tsx [hunk 3]
```diff
@@ -18,4 +18,35 @@ export const MainStyledTextField = styled(TextField, {
...
+        },
+    },
+    '& .MuiInputLabel-root': {
+        color: '#7E7E7E',
+        fontSize: '12px'
+    },
+    '& .MuiInputBase-input': {
+        color: '#7E7E7E',
+        backgroundColor: '#292a2b',
+        paddingTop: '15px',
```
Comment (mae1579) [inline src/components/StyledComponents/MainStyledTextField.tsx:42]: amazing css content 👍

File: src/common/ToggleButtonFilterStudents/ToggleButtonFilterStudents.tsx [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+    },
+}));
+
+export const ToggleButtonGroupFilterStudents = styled(ToggleButtonGroup)<ButtonProps>(() => ({
+    '& .Mui-selected': {
+        color: "#f7f7f7 !important",
+        borderBottom: '2px solid #E02735',
+        backgroundColor: '#292A2B !important',
+    },
+}));
```
Comment (mae1579) [inline src/common/ToggleButtonFilterStudents/ToggleButtonFilterStudents.tsx:25]: Keep doing that , all is great :)

Comment (patrycja-dev41171) [inline src/common/ToggleButtonFilterStudents/ToggleButtonFilterStudents.tsx:25]: Good use of components from Material UI.

**Review by mae1579 — COMMENTED** [review]
Huge changes all code is great 👍 , keep doing that good job 🥇

**Review by patrycja-dev41171 — COMMENTED** [review]
Great Job!

**Pull request merged**